src/PerfView/PerfViewData.cs

```diff
@@ -463,14 +463,24 @@ public static string ChangeExtension(string filePath, string newExtension)
             return Path.Combine(dirName, fileName);
         }
 
+        /// <summary>
+        /// for extensions of PerfView that don't have access to the GuiApp.MainWindow
+        /// </summary>
+        /// <param name="doAfter"></param>
+        public void Open(Action doAfter = null)
+        {
+            this.Open(GuiApp.MainWindow, GuiApp.MainWindow.StatusBar, doAfter);
+        }
+
```

Comment:
Do you actually use this call anymore (since you are now calling OpenWithoutWorker)?     If you are using it, great, just checking.

Reply:
no, not using any more.